routes/auth.js

```diff
@@ -75,7 +75,10 @@ if (auth.github.enabled) {
   passport.use(new passportGithub({ // eslint-disable-line new-cap
     clientID: auth.github.clientId,
     clientSecret: auth.github.clientSecret,
-    callbackURL: redirectURL
+    callbackURL: redirectURL,
+    authorizationURL: auth.github.authorizationURL,
+    tokenURL: auth.github.tokenURL,
+    userProfileURL: auth.github.userProfileURL
```

Comment:
What if those values are not provided in the config.yaml (they'll be set as undefined and this would probably cause issues)?

The config.js needs also to be updated to provide default values (empty strings?) for those.

Also, we could test for these values to be present before using them.

And finally, any configuration option needs also to be referenced in the README file.

Thanks :)